tasks/bake.js

```diff
@@ -27,7 +27,6 @@ module.exports = function( grunt ) {
 			basePath: "",
 			transforms: {},
 			parsePattern: /\{\{\s*([^\}]+)\s*\}\}/g,
-			transformGutter: "|",
```

Comment:
Let's add a check that if people set a transformGutter, we stop the bake task and print out an error message that `transformGutter` is no longer supported. We should maybe think about bumping major version, since that is technically what's necessary when removing a feature, as minor is only for depreciation. But I'm not sure we have to. I'm always a little bummed that I can't collect usage stats, to see what people do an don't use.